File: eclipse/src/listas/ListaGeneral.java
package listas;

public class ListaGeneral {

	public static void main(String[] args) {
		
		String[] paises = {"Venezuela", "Colombia", "Ecuador", "Peru","Chile"};
		
		//Formula General
		//Tipo[] nombre = {"Valor1","Valor2","Valor3"};
		
		System.out.println(paises[2]);
		
		System.out.println(paises[3]);
		
		//Error
		System.out.println(paises[4]);
		
		
		//Pais Chile
		paises[4] = "Argentina";
		System.out.println(paises[4]);
		
		System.out.println("-------------------------- FOR SIMPLE ---------------------");
		
		for (int i = 0; i < paises.length; i++) {
			System.out.println(paises[i]);
		}
		
		System.out.println("-------------------------- FOR MEJORADO ---------------------");
		
		// for(Tipo nombre : lista) {
		
		for(String pais_actual : paises) {
			System.out.println(pais_actual);
		}
	}

}

File: vscode/src_javascript/listas/ListaGeneral.js
let paises = ["Venezuela", "Colombia", "Ecuador", "Peru","Chile"];
		
//Formula General
//Tipo[] nombre = {"Valor1","Valor2","Valor3"};

console.log(paises[2]);

console.log(paises[3]);

//Error
console.log(paises[4]);


//Pais Chile
paises[4] = "Argentina";
console.log(paises[4]);

console.log("-------------------------- FOR SIMPLE ---------------------");

for (let i = 0; i < paises.length; i++) {
    console.log(paises[i]);
}

console.log("-------------------------- FOR MEJORADO ---------------------");

// for(Tipo nombre : lista) {

for(let pais_actual of paises) {
    console.log(pais_actual);
}